ui/packages/tsconfig/base.json

```diff
@@ -13,7 +13,10 @@
     "declarationMap": true,
     "esModuleInterop": true,
     "verbatimModuleSyntax": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "paths": {
+      "@inngest/components/*": ["../components/src/*"]
+    }
```

Comment:
This is needed for Turbo pack as per their docs. Before this change, the main `dashboard` app would be able to load the component (e.x. `Alert.tsx`) from the package (I believe due to the `package.json`'s `dependencies`), but it wouldn't be able to load the nested deps from within that component, e.g. `@inngest/components/utils/classNames` until this was added.